test/junit/scala/tools/nsc/interpreter/TabulatorTest.scala

```diff
@@ -11,27 +11,32 @@ case class VTabby(width: Int = 80, isAcross: Boolean = false, marginSize: Int =
 @RunWith(classOf[JUnit4])
 class TabulatorTest {
 
+  // à̧̄ is composed using 3 combining characters
+  val complexA = "a\u0300\u0304\u0327"
+  // 𝔟 is stored as 2 characters, but displayed as 1 grapheme
+  val frakturB = "\uD835\uDD1F"
```

Comment:
The random thing I tried was
```
scala> "a\u1ab0"
res6: String = a᪰
```
I asked my French guy, who insists on always using diacritics while his wife slacks, but he didn't know anything.

Reply:
The REPL doesn't seem to display combining characters properly. Printing them from a Scala app does work...

Reply:
But wait, I did that in REPL, where it worked. It's not displaying correctly in this Firefox on Windows. (I'm actually in the Ubuntu subsystem, where I just ssh'd to my real Ubuntu box and lost the arrow in zsh which I was just getting used to.)

Reply:
I'm running on MacOS, using Safari. Your a with double caret does not display correctly at all. Not in Safari, not when I paste "a\u1ab0" into the REPL, or use println("a\u1ab0") (both from REPL and in a unit test are wrong). So maybe Mac can't handle \u1ab0?

In the REPL, "a\u0300\u0304\u0327" is displayed as "a" with grave accent, followed by 2 combining characters. println("a\u0300\u0304\u0327") does the same thing in the REPL, but is correct from a unit test.

Reply:
Actually Safari also doesn't display "a\u0300\u0304\u0327" correctly; I don't know if these issues are important enough for a Scala bug...